grabbit/src/test/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessorSpec.groovy

```diff
@@ -29,7 +27,7 @@ import static com.twcable.jackalope.JcrConstants.NT_FILE
 import static javax.jcr.PropertyType.LONG
 import static javax.jcr.PropertyType.STRING
 
-@Subject(JcrNodesProcessor)
+//@Subject(JcrNodesProcessor)
```

Comment:
No need to remove, i guess i was trying something with test case that we were facing.